src/style.scss

```diff
@@ -221,6 +221,10 @@ select {
   }
 }
 
+.font-mono {
```

Comment:
This should be added in `tailwind.config.js` 
![image](https://user-images.githubusercontent.com/51686767/152375402-0f31f926-130b-48e4-bde0-5b17aad2c7c2.png)

Reply:
Great, moved to tailwind config now 🙇🏻